src/providers/OrganizationProvider.tsx

```diff
@@ -66,6 +67,17 @@ export default function OrganizationProvider({ children }: OrganizationProviderP
     await populateOrganizations();
   }, []);
 
+  const reloadAcceleratorOrganizations = useCallback(async () => {
+    const populateAcceleratorOrganizations = async () => {
+      const response = await OrganizationService.getAcceleratorOrganizations();
+      if (!response.error) {
+        setAcceleratorOrganizations(response.organizations);
+      }
+    };
+
+    await populateAcceleratorOrganizations();
+  }, []);
```

Comment:
I didn't add the ability to tracking the request status or pass a projectId and select an accelerator project because I wasn't sure if we needed it, but I can add it if we want either.